src/core/midi/MidiPort.cpp

```diff
@@ -169,6 +169,35 @@ void MidiPort::processOutEvent( const MidiEvent& event, const TimePos& time )
 			outEvent.setVelocity( fixedOutputVelocity() );
 		}
 
+		auto activePitch = m_activeMIDIPitch.find(event.key());
+		if( activePitch != m_activeMIDIPitch.end() )
+		{
+			outEvent.setKey( *activePitch );
+			
+			if( event.type() == MidiNoteOff )
+			{
+				m_activeMIDIPitch.remove( event.key() );
```

Comment:
Should be replaced with a call to erase, if we are going to use ``std::unordered_map``.

```suggestion
				m_activeMIDIPitch.erase( event.key() );
```